_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -3115,25 +3103,7 @@ public static function get_service_api_key( $request ) {
 	 * }
 	 */
 	public static function update_service_api_key( $request ) {
-		$service = self::validate_service_api_service( $request['service'] );
-		if ( ! $service ) {
-			return self::service_api_invalid_service_response();
-		}
-		$params     = $request->get_json_params();
-		$service_api_key    = trim( $params['service_api_key'] );
-		$option     = self::key_for_api_service( $service );
-		$validation = self::validate_service_api_key( $service_api_key, $service );
-		if ( ! $validation['status'] ) {
-			return new WP_Error( 'invalid_key', esc_html__( 'Invalid API Key', 'jetpack' ), array( 'status' => 404 ) );
-		}
-		$message = esc_html__( 'API key updated successfully.', 'jetpack' );
-		Jetpack_Options::update_option( $option, $service_api_key );
-		return array(
-			'code'            => 'success',
-			'service'         => $service,
-			'service_api_key' => Jetpack_Options::get_option( $option, '' ),
-			'message'         => $message,
-		);
+		return WPCOM_REST_API_V2_Endpoint_Service_API_Keys::update_service_api_key( $request ) ;
 	}
```

Comment:
Same as https://github.com/Automattic/jetpack/pull/10742/files#r237570229, but for `WPCOM_REST_API_V2_Endpoint_Service_API_Keys::update_service_api_key` (instead of ` __CLASS__ . '::get_service_api_key'`) on [R439](https://github.com/Automattic/jetpack/pull/10742/files#diff-58180905242554a749dbb58ba9464cc9R439)?